docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Managing Pull Requests
+---
+
+## Pull Requests and the Community
+
+Gatsby is an open source project. We have over 2,000 contributors and so much of what makes Gatsby great is contributed by folks like you!
```

Comment:
I'm not sure if this section is necessary... we should have some kind of intro though. Maybe replace this with a section describing who this doc is for and what you'll learn by reading it?

learnings could be something like - learn how to review changes being submitted to Gatsby or how your own contributions will be reviewed